ci/azure/linux.yml

```diff
@@ -54,3 +54,28 @@ jobs:
       inputs:
         testResultsFiles: 'test-data-*.xml'
         testRunTitle: 'Linux'
+    - powershell: |
```

Comment:
@alimcmaster1 any idea if there is a way to do this in bash?  xref #30393 the two remaining places where we do this are among the last windows-special-casing left

Reply:
Sure let me take a look - unless we are moving the CI to github actions soon? https://github.com/pandas-dev/pandas/pull/29715 @datapythonista in which case I might wait for that to happen.

Reply:
not so soon - things will move gradually and smooothly

Reply:
It will take time, and besides what Jeff said, it's not yet clear that GitHub will give us all the workers we need, so may be we need to keep Azure in the long term.